examples/spark/src/main/scala/org/apache/carbondata/examples/CaseClassDataFrameAPIExample.scala

```diff
@@ -42,8 +45,25 @@ object CaseClassDataFrameAPIExample {
       .mode(SaveMode.Overwrite)
       .save()
 
+    val employee = List(Employee("anubhav", 20000))
```

Comment:
@jackylk i added test cases in integration package for both spark version can you review?